src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -850,6 +851,8 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
                         ReportAnalyzerExecutionTime(consoleOutput, analyzerDriver, Culture, compilation.Options.ConcurrentBuild);
                     }
                 }
+
+                analyzerDriver?.Dispose();
```

Comment:
looks ok to me

---
In reply to: [121199682](https://github.com/dotnet/roslyn/pull/20111#discussion_r121199682) [](ancestors = 121199682)